pkg/cmd/roachtest/tests/jobs_util.go

```diff
@@ -191,6 +191,45 @@ func WaitForRunning(
 	}, maxWait)
 }
 
+type jobRecord struct {
+	status   string
+	payload  jobspb.Payload
+	progress jobspb.Progress
+}
+
+func (jr *jobRecord) GetHighWater() time.Time {
```

Comment:
i'm contemplating starting a dumb thread with cdc about the term "high water mark". They bind the _frontier_ of a span frontier to their persisted high water mark here:
https://github.com/msbutler/cockroach/blob/master/pkg/ccl/changefeedccl/changefeed_processors.go#L781

This confused me for several years. I feel like we should universally use the term low water mark for the lowest timestamp of a span frontier (i.e. the frontier), and the highest timestamp in the frontier as the high water mark. I think clearing this up will help everyone out, especially the new hires.

But maybe i'm missing some context.